client/src/components/video.vue

```diff
@@ -678,6 +681,46 @@
         }, 100)
       }
     }
+    onTouchHandler(event: TouchEvent) {
+      let touches = event.changedTouches
+      let first = touches[0]
+      let type = ''
+      switch (event.type) {
+        case 'touchstart':
+          type = 'mousedown'
+          break
+        case 'touchmove':
+          type = 'mousemove'
+          break
+        case 'touchend':
+          type = 'mouseup'
+          break
+        default:
+          return
+      }
+
+      const simulatedEvent = document.createEvent('MouseEvent')
+      simulatedEvent.initMouseEvent(
```

Comment:
This event seems to be [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/initMouseEvent).

Reply:
This seems to be replacement:

```bash
var mouseEvent = new MouseEvent('mousedown', {
  clientX: touch.clientX,
  clientY: touch.clientY
});
```